app/helpers/proof_of_recognition_helper.rb

```diff
@@ -1,44 +1,54 @@
 module ProofOfRecognitionHelper
+  include RegionHelper
+
   def proof_of_recognition_requirements_for(region:)
     if region.status_check_written? && region.sanction_check_written?
       return(
-        written_status_reasons.insert(2, *written_sanction_reasons(region)) +
-          ["that you’re qualified to teach at state or government schools"]
+        written_status_reasons(region).insert(
+          2,
+          *written_sanction_reasons(region),
+        )
       )
     end
 
-    return written_status_reasons if region.status_check_written?
+    return written_status_reasons(region) if region.status_check_written?
     return written_sanction_reasons(region) if region.sanction_check_written?
     []
   end
 
   def proof_of_recognition_description_for(region:)
     if region.teaching_authority_provides_written_statement?
       "The document must confirm:"
-    elsif region.status_check_written?
-      "The authority or territory that recognises you as a teacher must confirm:"
-    elsif region.sanction_check_written?
-      "The education department or authority must also confirm in writing:"
+    else
+      "In the #{region_certificate_name(region)} the #{region_teaching_authority_name(region)} must confirm " \
+        "that your authorisation to teach has never been:"
     end
   end
 
   private
 
-  def written_status_reasons
-    [
-      "that you've completed a teaching qualification/teacher training",
-      "that you’ve successfully completed any period of professional experience " \
-        "comparable to an induction period (if required)",
-      "the age ranges and subjects you’re qualified to teach",
-    ]
+  def written_status_reasons(region)
+    if region.teaching_authority_provides_written_statement?
+      [
+        "that you've completed a teaching qualification/teacher training",
+        "that you’ve successfully completed any period of professional experience comparable to an induction period" \
+          "(if required)",
+        "the age ranges and subjects you’re qualified to teach",
+        "that you’re qualified to teach at state or government schools",
+      ]
+    else
+      %w[suspended barred cancelled revoked]
```

Comment:
Reading through this list again it occurs to me that this seems more related to sanctions than status, i.e. we're losing reasons that relate to qualification or age ranges, I wonder if it's worth double checking with Wes/Dom if this is right?